ingest-app/docs/api.md

```diff
@@ -22,9 +22,6 @@ Join the [CMR Client Developer Forum](https://wiki.earthdata.nasa.gov/display/CM
   * /providers/\<provider-id>/services/\<native-id>
     * [PUT - Create or update a service.](#create-update-service)
     * [DELETE - Delete a service.](#delete-service)
-  * /providers/\<provider-id>/subscriptions/\<native-id>
```

Comment:
Rebase / copy paste gone wrong or are you intending to delete subscription API documentation notes with this PR?